source/postprocess/visualization/particle_count.cc

```diff
@@ -43,8 +43,8 @@ namespace aspect
                                "active postprocessors. You need to select this postprocessor to "
                                "be able to select the <particle count> visualization plugin."));
 
-        const std::multimap<aspect::Particle::types::LevelInd, aspect::Particle::Particle<dim> > particles =
-          tracer_postprocessor->get_particle_world().get_particles();
+        const std::multimap<aspect::Particle::types::LevelInd, aspect::Particle::Particle<dim> > *particles =
+          &tracer_postprocessor->get_particle_world().get_particles();
```

Comment:
:-)

Why not make the variable a reference instead? Then you won't need the change below. In fact, since you only need this variable in one place below, you could also just inline the call to the function in the one place you need it.